docs/security/Staticfile.auth

```diff
@@ -1 +0,0 @@
-ssp_viewer:$apr1$/O6mfuN.$pct4cTe8r1j2B5RjM9mDZ1
```

Comment:
Did this come from a private repo by mistake or something? Why were we keeping a basic auth username and (hashed) password in this repo to protect access to public contents?

Reply:
This wasn't a mistake. It was there to build out the compliance masonry with `gitpages` and push out as a cloud.gov instance. I initially created with the hopes of using compliance masonry.  These docs are super out of date and can be removed since we didn't go this route.